### PR TITLE
Update BATT.yaml

### PR DESCRIPTION
**Pull request by Karthi8883**
updated points for battery monitoring
 - max_battery_temperature_sensor
  - max_battery_voltage_sensor
  - min_battery_temperature_sensor
  - min_battery_voltage_sensor
  - battery_charge_status
  - battery_failed_alarm

### CODE REVIEW

**Comment by tasodorff** [comment]
validator errors

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -753,6 +753,13 @@ literals:
   - DISCHARGING
   - STANDBY
 - battery_charge_percentage_sensor
+- max_battery_temperature_sensor
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:756]: `max` and `min` would require an `aggregation_descriptor` in this use case -- I am assuming you are trying to model the high and low temperature and voltage over a period of time?

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:756]: There are multiple internal temperature sensors and voltage sensors in the system. They are only providing the min and max values among all the sensors measured.

Comment (cstirdivant) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:756]: thank you for clarifying :)

**Review by cstirdivant — APPROVED** [review]

**Comment by mschulze17** [comment]
@cstirdivant yours to merge :)

**Pull request merged**